app/admin/components/TextCard.jsx

```diff
@@ -102,6 +107,7 @@ class TextCard extends Component {
             onSave={this.save.bind(this)}
           />
         </Dialog>
+        { this.props.type === "story_footnote" && <div>Use Footnote Reference: <strong>{`Foot{{${this.props.ordering + 1}}}`}</strong></div>}
```

Comment:
You already have `type` from `this.props` on line 81, and you should also grab `ordering` as well.